source/renderer/app/api/addresses/types.js

```diff
@@ -1,22 +1,29 @@
 // @flow
 import WalletAddress from '../../domains/WalletAddress';
 
-export type Address = {
+export type AddressState = 'used' | 'unused';
+
+export type GetAddressesRequestQueryParams = {
+  state: AddressState,
+};
+
+export type AdaAddress = {
```

Comment:
@MarcusHurney with that I want avoid confusion and tell in code that type is with Ada prefix because response come from api with one set of properties which are different regarding to WalletAddress declaration in domains. If you think that prefix is not necessary I can remove? Now you know what the point was